mne/viz/topo.py

```diff
@@ -521,6 +521,12 @@ def _plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
     merge_grads : bool
         Whether to use RMS value of gradiometer pairs. Only works for Neuromag
         data. Defaults to False.
+    legend : bool | int | string | tuple
+        If True, create a legend based on evoked.comment. If False, disable the
+        legend. Otherwise the legend is created and the parameter value is
+        passed as the location parameter to the matplotlib legend call. For 
+        example, 'upper right'. By default, the location is chosen
+        automatically.
```

Comment:
From http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.legend:

"Alternatively can be a 2-tuple giving x, y of the lower-left corner of the legend in axes coordinates ..."